Packs/Code42/Playbooks/playbook-Code42_File_Download.yml

```diff
@@ -21,6 +21,11 @@ inputs:
       root: File
       transformers:
       - operator: uniq
+- description: The name of the file to save as.
```

Comment:
Similar to some other conversations in slack, I'm questioning the value of having a playbook that does virtually the same thing as a command, but we can discuss more offline.